showcase/build/iaf_cond_alpha_implicit.h

```diff
@@ -0,0 +1,888 @@
+  *
+  * These variables must be initialized by @c calibrate, which is called before
+  * the first call to @c update() upon each call to @c Simulate.
+  * @node Variables_ needs neither constructor, copy constructor or assignment
+  * operator,
+  *       since it is initialized by @c calibrate(). If Variables_ has members
+  * that
+  *       cannot destroy themselves, Variables_ will need a destructor.
+  */
+  struct Variables_ {
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double PSConInit_E; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    double PSConInit_I; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    long RefractoryCounts; //
+
+    /* generated by template org.nest.nestml.neuron.function.MemberDeclaration*/
+
+    long r; //
+  };
+
+  /**
+    * Buffers of the neuron.
+    * Ususally buffers for incoming spikes and data logged for analog recorders.
+    * Buffers must be initialized by @c init_buffers_(), which is called before
+    * @c calibrate() on the first call to @c Simulate after the start of NEST,
+    * ResetKernel or ResetNetwork.
+    * @node Buffers_ needs neither constructor, copy constructor or assignment
+   * operator,
+    *       since it is initialized by @c init_nodes_(). If Buffers_ has members
+   * that
+    *       cannot destroy themselves, Buffers_ will need a destructor.
+    */
+  struct Buffers_ {
+    Buffers_(iaf_cond_alpha_implicit &);
+    Buffers_(const Buffers_ &, iaf_cond_alpha_implicit &);
+
+    /** Logger for all analog data */
+    nest::UniversalDataLogger<iaf_cond_alpha_implicit> logger_;
+
+    inline nest::RingBuffer &get_spikeInh() { return spikeInh; }
+    nest::RingBuffer spikeInh;
+    //!< Buffer incoming Buffers through delay, as sum
+    ;
+    double spikeInh_last_value_;
+    inline nest::RingBuffer &get_spikeExc() { return spikeExc; }
+    nest::RingBuffer spikeExc;
+    //!< Buffer incoming Buffers through delay, as sum
+    ;
+    double spikeExc_last_value_;
+
+    nest::RingBuffer currents;
+    //!< Buffer incoming Buffers through delay, as sum
+    ;
+    inline nest::RingBuffer &get_currents() { return currents; }
+    double currents_last_value_;
+
+    /** GSL ODE stuff */
+    gsl_odeiv_step *__s;    //!< stepping function
+    gsl_odeiv_control *__c; //!< adaptive stepsize control function
+    gsl_odeiv_evolve *__e;  //!< evolution function
+    gsl_odeiv_system __sys; //!< struct describing system
+
+    // IntergrationStep_ should be reset with the neuron on ResetNetwork,
+    // but remain unchanged during calibration. Since it is initialized with
+    // step_, and the resolution cannot change after nodes have been created,
+    // it is safe to place both here.
+    double __step; //!< step size in ms
+    double
+        __integration_step; //!< current integration time step, updated by GSL
+  };
+
+  /* generated by template
```

Comment:
same comment as before: I thinks it would be good to encapsulate the region generated by one function.
Can you do something like

    /* begin region generated by org.nest.nestml.a.given.function */
    ...
    a lot of code
    ...
    /* end region generated by org.nest.nestml.a.given.function */